pages/index.js

```diff
@@ -61,6 +61,15 @@ export default function Home({ people, preview }) {
   )
 }
 
+// Dry up h2 styling
+function Subhead({ children }) {
+  return (
+    <h2 className='text-2xl mt-8 font-bold'>
```

Comment:
@isaazgarcia curious your thoughts about this.  I set that `mt-8` here, thinking it would override the `space-y-4` from the `<main>` container.  But instead, the `space-y-4` is taking precedence.  I wanted to make these `h2`s have greater margins up top.  Is this the kinda case where I'd break out of TW or is there some way to affect the precedence?

![Homepage 2023-03-23 at 4 15 22 PM](https://user-images.githubusercontent.com/77567/227385110-88bd76aa-99c8-4abc-9a53-01f34fc1dfc8.jpg)